docs/v1/P4Runtime-Spec.mdk

```diff
@@ -3043,6 +3049,10 @@ limitation. It is recommended that, for the sake of portability, P4Runtime
 clients do not try to insert additional entries once the size indicated in
 P4Info has been reached.
 
+The `is_const` field must be `false` in any `INSERT`, `MODIFY`, or
```

Comment:
There is a comprehensive list of fields for the `TableEntry` message earlier in this section (or at least it should be a comprehensive list...). I have noticed that `is_const` is not listed there.

Look for:
> P4Runtime supports inserting, modifying, deleting and reading table entries with
the `TableEntry` entity, which has the following fields:

Reply:
Added is_const field to that list of fields.